tiledb/api/c_api/profile/profile_api_experimental.h

```diff
@@ -190,6 +190,27 @@ TILEDB_EXPORT capi_return_t tiledb_profile_get_param(
 TILEDB_EXPORT capi_return_t tiledb_profile_save(
     tiledb_profile_t* profile, tiledb_error_t** error) TILEDB_NOEXCEPT;
 
+/**
+ * Loads a profile from the local file.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * tiledb_profile_t* profile;
+ * tiledb_error_t* error = NULL;
+ * tiledb_profile_alloc(&profile, &error);
+ * tiledb_profile_load(profile, &error);
```

Comment:
Can we add a TODO to make this more meaningful when we have a way to set that profile to a config so that we show to the user how to use it after loading it?

Reply:
So I was confused about how a loaded profile can be used, let's remove the TODO an just show that you can do `get_parameter` and `dump`. I think those are the only operations supported so far.